tests/test_classes.py

```diff
@@ -129,7 +129,8 @@ def test06_reference_internal(clean):
 
 def test07_big():
     x = [t.Big() for i in range(1024)]
-    x = [t.BigAligned() for i in range(1024)]
+    x2 = [t.BigAligned() for i in range(1024)]
+    assert x == x2
```

Comment:
FYI, not sure what was intended here. Guessed maybe this?

Reply:
Apparently that is not what was intended here. Is this just supposed to run and not check anything?

Reply:
```suggestion
```